packages/paste-website/src/components/site-wrapper/index.tsx

```diff
@@ -20,6 +20,14 @@ const globalStyles = css`
     margin: 0;
     font-size: 14px;
   }
+
+  iframe {
```

Comment:
I added this here for now as global style. It makes iframe embeds actually work with Aspect Ratio. We should probably just move this to the default styles for Aspect Ratio. Thoughts?

Reply:
I don't like this as a global style, it doesn't scale to non-AspectRatio iframes.

I think you just have to make a new component and put the styles there.

Reply:
hmmm I feel like the expectation from reading the docs is that it should "just work", so maybe it belongs to the Aspect Ratio. Do you need to do the same thing for an image? What's the least amount of work for a consumer to do for all cases?

Reply:
It's probably safe to do something like `> :first-child` here.

Reply:
@SiTaggart you're right. It should "just work". I'm going to move that style over to the Aspect Ratio where it should be. I think if that component was kicked off properly, we would have caught that. Image is much simpler:

```
<AspectRatio ratio="4:3">
    <img src="" />
</AspectRatio>
```

Reply:
Added embed, iframe, object, video styles to Aspect Ration in the latest commit.